### PR TITLE
PR feature 09/hello-javascript

### PR DESCRIPTION
**Pull request by adamkarb**

### CODE REVIEW

**Comment by adamkarb** [comment]
[WIP Issue](https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/125)

**Comment by johnalxndr** [comment]
:+1:

**Comment by mfees** [comment]
:+1:

**Comment by jgarcia99** [comment]
:+1:

**Comment by al-the-x** [comment]
More commits, help you they will. Yes.

File: doc/JS.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Primitive: Boolean]
+
+* _symbol_: `boolean`
+* _pronunciation_: "boo-lee-in"
```
Comment (al-the-x) [inline doc/JS.md]: You're very funny. I just LOL'd on the train. Very embarassing.

File: doc/JS.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Primitive: Boolean]
+
+* _symbol_: `boolean`
```
Comment (al-the-x) [inline doc/JS.md]: More like `true` and `false`...

File: doc/JS.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
```
Comment (al-the-x) [inline doc/JS.md]: What, you can't come up with any of your own?

File: doc/JS.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+### [Primitive: Boolean]
```
Comment (al-the-x) [inline doc/JS.md]: Maybe should be a link like the examples?

File: doc/JS.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+### [Number]
+
+* _symbol_: `n/a`
+* _pronunciation_: Number
+* _examples_:
+```javascript
+1 2 4 6 7 9
+```
```
Comment (al-the-x) [inline doc/JS.md]: Now, now. What _symbols_ do we use to represent the `Number` type? How do you write them _literally_...? Y'know, as in your examples, but are there _more_? Hint: there are.

File: doc/JS.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+### [String]
+
+* _symbol_: `Anything between single or double quotes`
+* _pronunciation_: string
+* _examples_:
+```javascript
+"This is a string of seven words"
+```
```
Comment (al-the-x) [inline doc/JS.md]: Looking more for the symbols used to write a literal, e.g. `'...'`, `"..."`, and some examples of said literals being used. Maybe throw some escape characters in there... show both versions of quotes in use... that kinda stuff.

File: doc/JS.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+### [Infinity]
+
+* _symbol_: `sideways 8`
+* _pronunciation_: in-fin-it-ee
+* _examples_:
+```javascript
+6 + infinity === infinity
+```
```
Comment (al-the-x) [inline doc/JS.md]: So the _symbol_ for this built-in value is `Infinity` (case is important). Try to type "sideways 8" on a standard keyboard, I dare you.

File: doc/JS.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+### [NaN = Not a Number]
+
+* _symbol_: `NaN`
+* _pronunciation_: not a number
+* _examples_:
+```javascript
+3 / 0 === NaN
+```
```
Comment (al-the-x) [inline doc/JS.md]: You should try this on the REPL... I bet you'll be surprised. Stick it in an `assert()` while you're at it.

File: doc/JS.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+### [Undefined]
+
+* _symbol_: `Undefined`
+* _pronunciation_: un-de-fyned
+* _examples_:
+```javascript
+var dog = 
+
+function(dog) = variable undefined
+```
```
Comment (al-the-x) [inline doc/JS.md]: This is all kinds of syntax errors. Run your examples through the interpreter!

**Comment by al-the-x** [comment]
Deviated a little from the assignment... No operators. Despite being checked on your _WIP Issue_... What you tryin' to pull, Karberator?

**Pull request merged**